Packs/AzureSQLManagement/ReleaseNotes/1_1_18.md

```diff
@@ -0,0 +1,39 @@
+
+#### Integrations
+
+##### Azure SQL Management
+
+- Updated the integration and took it out of beta.
+
+###### Command: 'azure-sql-db-list'
+- Added 2 new arguments 'subscription_id', 'resource_group_name' that if given will override the parameters that was given in the pack configuration. 
```

Comment:
Please use the release notes template.
Group all changes together.
For example:
-  Added 2 new arguments *subscription_id* and *resource_group_name* that if given will override the parameters that were given in the pack configuration for the following commands:
  - list the commands